### DISCUSSION TITLE
mspider/user-guide/traffic-governance/virtual-service/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1274] [Announcements]
# mspider/user-guide/traffic-governance/virtual-service/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/traffic-governance/virtual-service/

**Comment by liyuerich** [comment]
几个虚拟服务的服务列表的截图不一致，有的有规则标签，有个没有

**Comment by windsonsea** [comment]
已提 PR 修改：
https://github.com/DaoCloud/DaoCloud-docs/pull/1278

**Comment by windsonsea** [comment]
整页涉及`规则标签`的插图已一致，请刷新页面：https://docs.daocloud.io/mspider/user-guide/traffic-governance/virtual-service/

所以这个先 close 了额

**Comment by Adityayxt** [comment]
虚拟服务，创建规则时，重试条件需要文档介绍，不知道这些条件的作用是什么，能起到什么效果
